Chatkit/Redux/Store/Store.swift

```diff
@@ -0,0 +1,110 @@
+
+enum Action {
+    case subscriptionEvent(Wire.Event.EventType)
+    case received(user: Wire.User)
+    case fetching(userWithIdentifier: String)
+}
+
+extension Action: Equatable {}
+
+protocol StoreDelegate: AnyObject {
```

Comment:
Is there any reason for this protocol to inherit from `AnyObject` instead of `class`?

Reply:
`AnyObject` is preferred over `class` from Swift 4.1 onwards but I'm not fully aware of the reasoning.

Something to do with this...
https://github.com/apple/swift-evolution/blob/master/proposals/0156-subclass-existentials.md

Also this...
https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#anyObjectProtocol